pytest test_convert_notebook_to_html fails due to SHA256 error

in https://github.com/elyra-ai/kfp-notebook/blob/master/etc/tests/test_bootstrapper.py#L332

```
assert hs.fileChecksum(notebook_output_html_file, "sha256") == HTML_SHA256
E           AssertionError: assert '346a224c213d...ad9ddff2c21ce' == '4f717d3bbb41...0b6109846d509'
E             - 4f717d3bbb41cb7b7d03814dee6639d3190e5b80f8a80b9af310b6109846d509
E             + 346a224c213da52f704af75fa5166e0f19222aedbdbc2616363ad9ddff2c21ce
```

The test works as expected on macos but fails when run as art of the github ci/cd

After getting some clues from Patrick, and since it smelled like a transitive dependency issue (because if first crept up on a doc-only PR), I narrowed this down to updates in four dependencies different from successful tests to failing tests:

Successful test dependencies:
```
nest-asyncio-1.4.1 pandocfilters-1.4.2 pygments-2.7.1 regex-2020.10.22 tqdm-4.50.2
```
Failing test dependencies:
```
nest-asyncio-1.4.2 pandocfilters-1.4.3 pygments-2.7.2 regex-2020.10.23 tqdm-4.51.0
```

with the winner being the `pgyments` package:
```
$ pytest etc/tests/test_bootstrapper.py::test_convert_notebook_to_html
============================================================================ test session starts =============================================================================
platform darwin -- Python 3.6.11, pytest-6.1.1, py-1.9.0, pluggy-0.13.1
rootdir: /Users/kbates/repos/oss/elyra-ai/kfp-notebook
plugins: tornasync-0.6.0.post2, shutil-1.7.0, virtualenv-1.7.0, console-scripts-1.0.0
collected 1 item                                                                                                                                                             

etc/tests/test_bootstrapper.py .                                                                                                                                       [100%]

============================================================================= 1 passed in 1.80s ==============================================================================
~/repos/oss/elyra-ai/kfp-notebook (master)$ 
~/repos/oss/elyra-ai/kfp-notebook (master)$ pip install Pygments==2.7.2
Looking in indexes: https://pypi.org/simple/
Collecting Pygments==2.7.2
  Using cached Pygments-2.7.2-py3-none-any.whl (948 kB)
Installing collected packages: Pygments
  Attempting uninstall: Pygments
    Found existing installation: Pygments 2.7.1
    Uninstalling Pygments-2.7.1:
      Successfully uninstalled Pygments-2.7.1
Successfully installed Pygments-2.7.2
~/repos/oss/elyra-ai/kfp-notebook (master)$ pytest etc/tests/test_bootstrapper.py::test_convert_notebook_to_html
============================================================================ test session starts =============================================================================
platform darwin -- Python 3.6.11, pytest-6.1.1, py-1.9.0, pluggy-0.13.1
rootdir: /Users/kbates/repos/oss/elyra-ai/kfp-notebook
plugins: tornasync-0.6.0.post2, shutil-1.7.0, virtualenv-1.7.0, console-scripts-1.0.0
collected 1 item                                                                                                                                                             

etc/tests/test_bootstrapper.py F                                                                                                                                       [100%]

================================================================================== FAILURES ==================================================================================
_______________________________________________________________________ test_convert_notebook_to_html ________________________________________________________________________

tmpdir = local('/private/var/folders/4w/qxgsbhyx1y93qlk9mnzwjryc0000gn/T/pytest-of-kbates/pytest-56/test_convert_notebook_to_html0')

    def test_convert_notebook_to_html(tmpdir):
        notebook_file = os.getcwd() + "/etc/tests/resources/test-notebookA.ipynb"
        notebook_output_html_file = "test-notebookA.html"
    
        with tmpdir.as_cwd():
            bootstrapper.NotebookFileOp.convert_notebook_to_html(notebook_file, notebook_output_html_file)
    
            assert os.path.isfile(notebook_output_html_file)
>           assert hs.fileChecksum(notebook_output_html_file, "sha256") == HTML_SHA256
E           AssertionError: assert '346a224c213d...ad9ddff2c21ce' == '4f717d3bbb41...0b6109846d509'
E             - 4f717d3bbb41cb7b7d03814dee6639d3190e5b80f8a80b9af310b6109846d509
E             + 346a224c213da52f704af75fa5166e0f19222aedbdbc2616363ad9ddff2c21ce

etc/tests/test_bootstrapper.py:332: AssertionError
========================================================================== short test summary info ===========================================================================
FAILED etc/tests/test_bootstrapper.py::test_convert_notebook_to_html - AssertionError: assert '346a224c213d...ad9ddff2c21ce' == '4f717d3bbb41...0b6109846d509'
============================================================================= 1 failed in 2.01s ==============================================================================
```

--- a/etc/tests/test_bootstrapper.py
+++ b/etc/tests/test_bootstrapper.py
@@ -37,12 +37,7 @@ import bootstrapper
 # NOTE: Any changes to etc/tests/resources/test-notebookA.ipynb require an
 # update of etc/tests/resources/test-archive.tgz  using the command below:
 # tar -cvzf test-archive.tgz test-notebookA.ipynb
-#
-# There is also a need to update the following SHA256 hash used in
-# test_convert_notebook_to_html() below.
-#
 
-HTML_SHA256 = '4f717d3bbb41cb7b7d03814dee6639d3190e5b80f8a80b9af310b6109846d509'
 
 MINIO_HOST_PORT = os.getenv("MINIO_HOST_PORT", "127.0.0.1:9000")
 
@@ -329,7 +324,12 @@ def test_convert_notebook_to_html(tmpdir):
         bootstrapper.NotebookFileOp.convert_notebook_to_html(notebook_file, notebook_output_html_file)
 
         assert os.path.isfile(notebook_output_html_file)
-        assert hs.fileChecksum(notebook_output_html_file, "sha256") == HTML_SHA256
+        # Validate that an html file got generated from the notebook
+        with open(notebook_output_html_file, 'r') as html_file:
+            html_data = html_file.read()
+            assert html_data.startswith("<!DOCTYPE html>")
+            assert "&quot;TEST_ENV_VAR1&quot;" in html_data   # from os.getenv("TEST_ENV_VAR1")
+            assert html_data.endswith("</html>\n")
 
 
 def test_fail_convert_notebook_to_html(tmpdir):

